generator.ts

```diff
@@ -128,8 +128,16 @@ export async function generate(token: string, configYaml: string): Promise<Repor
         continue
       }
 
-      const defaultStages = ['Proposed', 'Accepted', 'Blocked', 'In-Progress', 'Done', 'Unmapped']
-      for (const phase of defaultStages) {
+      const validStages = ['Proposed', 'Accepted', 'Blocked', 'In-Progress', 'Done', 'Unmapped']
```

Comment:
We don't want to doc unmapped as that stage is meant for issues that aren't mapped properly.  I left blocked out of the docs even though you can technically map it since the how we work tooling guidance instructs folks to add a Blocked column.